srcpkgs/python3-txtorcon/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'python3-txtorcon'
 pkgname=python3-txtorcon
-version=21.1.0
-revision=4
+version=22.0.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3 python3-setuptools"
 depends="lsof python3-automat python3-cryptography python3-six python3-Twisted
```

Comment:
https://github.com/meejah/txtorcon/blob/0c416cc8fe18b913cd0c7422935885a1bfecf4c0/txtorcon/util.py#L122
```suggestion
depends="lsof which python3-automat python3-cryptography python3-six python3-Twisted
```

Reply:
Isn't `which` part of base-{minimal,voidstrap,system}?

Reply:
Yes, but Debian is pushing programs to remove their which dependencies and I think it's nice to mention explicitly.